src/main/webapp/chat.js

```diff
@@ -0,0 +1,18 @@
+function init() {
+    // Calls the updateChat servlet every second for new messages
+    setInterval(function() {
```

Comment:
Can this be written as an arrow function or does the second parameter block that?